Procurement/Controls/FragmentStashViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using POEApi.Model;
+using Procurement.ViewModel;
+
+namespace Procurement.Controls
```

Comment:
Add a unit test, similar to, say, `GetShardCurrencyStashTest`, to make sure we correctly do or do not find items that do or do not, respectively, exist in the fragment tab.  You can use the bin file I included in #857 in the test.

Reply:
Resolved.